packages/grid/_modules_/grid/hooks/features/groupingColumns/gridGroupingColumnsUtils.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  GridColDef,
+  GridRowId,
+  GridRowTreeConfig,
+  GridRowTreeNodeConfig,
+  GridStateColDef,
+  GridValueGetterSimpleParams,
+} from '../../../models';
+import { GridFilterState } from '../filter';
+import { GridComponentProps } from '../../../GridComponentProps';
+
+export const GROUPING_COLUMN_SINGLE = '__row_group_by_columns_group__';
+
+export const GROUPING_COLUMNS_FEATURE_NAME = 'grouping-columns';
+
+export const getGroupingColDefFieldFromGroupingCriteriaField = (
```

Comment:
Should we export this utility to avoid users hard-coding there grouping column field for `sortModel` and `filterModel` ?